packages/core/src/components/DismissableBanner/DismissableBanner.tsx

```diff
@@ -88,9 +103,17 @@ export const DismissableBanner: FC<Props> = ({ variant, message, id }) => {
 
   return (
     <Snackbar
-      anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+      anchorOrigin={
+        fixed
+          ? { vertical: 'bottom', horizontal: 'center' }
+          : { vertical: 'top', horizontal: 'center' }
+      }
       open={!dismissedBanners.has(id)}
-      classes={{ root: classes.root }}
+      classes={
+        fixed
+          ? { root: classes.root }
+          : { root: classNames(classes.topPosition, classes.root) }
+      }
```

Comment:
This could be shortened to `classes={{ root: classNames(classes.root, fixed && classes.topPosition) }}` if you want, since `classNames` skips over anything that's falsy. But it's a matter of taste, yours is arguably clearer to read.